articles/101_build_tool.md

```diff
@@ -40,23 +31,23 @@ Such a workflow is impracticable at scale without a tool that automates that pro
 
 A build tool performs the task of building a set of packages with a single invocation.
 For ROS 1 multiple different tools provide support for this, namely `catkin_make`, `catkin_make_isolated`, and `catkin_tools`.
-For ROS 2 the build tool providing this functionality is called `ament_tools`.
+For ROS 2 up to the Ardent release the build tool providing this functionality is called `ament_tools`.
 
 This article describes the steps to unify these build tools as well as extend the field of application.
 
 ## Goal
 
 The goal of a unified build tool is to build a set of packages with a single invocation.
 It should work with ROS 1 packages as well as ROS 2 packages which provide the necessary information in their manifest files.
-It should also work with packages that do not provide manifest files themselves, given that the meta information is externally provided.
-This will allow the build tool to be utilized for non-ROS packages (e.g. Gazebo including its dependencies).
+It should also work with packages that do not provide manifest files themselves, given that the necessary meta information is provided externally.
+This will allow the build tool to be utilized for non-ROS packages (e.g. Gazebo including its ignition dependencies).
```

Comment:
Why is this now restricted to ignition deps? this wouldnt work for e.g. sdformat?

Reply:
It is not "restricted". It is just an example:

> e.g. ...